examples/swarm-setup.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	metrics "github.com/ipfs/go-libp2p/p2p/metrics"
+	tu "github.com/ipfs/go-libp2p/testutil"
+	ma "github.com/jbenet/go-multiaddr"
+	swarm "go-libp2p/p2p/net/swarm"
```

Comment:
Trying to understand why for swarm and peer it didn't complain that I didn't add `github.com/ipfs` to the path